0022_gcc4.4_preprocessor_syntax.dpatch by  <fabbe@paniq.net>

The #elif preprocessor directive requires a test condition.
GCC 4.4 enforces this rule.

diff -urNad trunk~/plugins/digestmd5.c trunk/plugins/digestmd5.c
--- trunk~/plugins/digestmd5.c	2006-06-04 12:26:19.000000000 +0300
+++ trunk/plugins/digestmd5.c	2009-01-26 13:29:40.000000000 +0200
@@ -2715,7 +2715,7 @@
 	"DIGEST-MD5",			/* mech_name */
 #ifdef WITH_RC4
 	128,				/* max_ssf */
-#elif WITH_DES
+#elif defined(WITH_DES)
 	112,
 #else 
 	1,
@@ -4034,7 +4034,7 @@
 	"DIGEST-MD5",
 #ifdef WITH_RC4				/* mech_name */
 	128,				/* max ssf */
-#elif WITH_DES
+#elif defined(WITH_DES)
 	112,
 #else
 	1,
